### PR TITLE
Release 2.5.0

### PR DESCRIPTION
**Pull request by jtslear**
* closes waffleio/waffle.io-takeout#190

### CODE REVIEW

File: releases/2.x.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# Waffle Takeout 2.5
+
+## Waffle Takeout 2.5.0
+_Released 2016.09.26_
+
+
```
Comment (songawee) [inline releases/2.x.md]: Please remove this empty line

File: releases/2.x.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+#### Maintenance Release
```
Comment (songawee) [inline releases/2.x.md]: I think we've been starting to follow more of the conventional changelog - similar to https://github.com/btford/grunt-conventional-changelog/blob/master/CHANGELOG.md. Can you please change this title to be one of these types? - https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#type
Perhaps refactor?

**Comment by jtslear** [comment]
@songawee can i get a re-review?

**Comment by songawee** [comment]
👍

**Pull request merged**